src/encoded/static/components/browse/components/above-table-controls/SelectedFilesControls.js

```diff
@@ -58,14 +62,14 @@ export class SelectAllFilesButton extends React.PureComponent {
         const { totalFilesCount, selectedFiles } = this.props;
         if (!totalFilesCount) return false;
         // totalFilesCount as returned from bar plot aggs at moment is unique.
-        if (totalFilesCount === uniqueFileCount(selectedFiles)){
+        if (totalFilesCount === this.memoized.uniqueFileCount(selectedFiles)){
             return true;
         }
         return false;
     }
 
     handleSelectAll(evt){
-        const { selectFile, resetSelectedFiles, href } = this.props;
+        const { selectFile, selectedFiles, resetSelectedFiles, href, context, totalFilesCount } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['selectedFiles' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4660599)